### PR TITLE
0.6 test

### PR DESCRIPTION
**Pull request by sakows**
Update main README and sidecar installation scripts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,28 @@
 # NGINX Service Mesh with Istio compatibility
-This repository provides an implementation of a NGINX based service mesh (NGINMESH).  NGINMESH is compatible with Istio.  It leverages NGINX as sidecar proxy. 
+This repository provides an implementation of a NGINX based service mesh (nginmesh).  Nginmesh is compatible with Istio.  It leverages NGINX as sidecar proxy.
```
Comment (sehz) [inline README.md]: use following convention from now: 'nginMesh'.

File: README.md [hunk 2]
```diff
@@ -71,46 +69,66 @@ kubectl get pods -n istio-system
...
 
-### Deploy Kafka
-
-Kafka deployment use Helm.  
-
-1. Set up Helm.  Please follow Helm setup guide:
+7. Verify that istio-injection label is not labeled for the default namespace :
 ```
-https://docs.helm.sh/using_helm/#quickstart
+kubectl get namespace -L istio-injection
```
Comment (sehz) [inline README.md:77]: move this to sidecar section

File: README.md [hunk 3]
```diff
@@ -71,46 +69,66 @@ kubectl get pods -n istio-system
...
 ```
-If you have RBAC issue, please run following scripts
 ```
- kubectl create serviceaccount --namespace kube-system tiller
- kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
- kubectl patch deploy --namespace kube-system tiller-deploy -p '{"spec":{"template":{"spec":{"serviceAccount":"tiller"}}}}' 
- ```
-2. Run following script to setup Kafka. It is installed in 'kafka' namespace.  It is possible to use your existing kafka installation.
+NAME           STATUS        AGE       ISTIO-INJECTION
```
Comment (sehz) [inline README.md:80]: seems like this should belong in the Istio installation

File: README.md (hunk 1 above)
Comment (sakows) [reply README.md]: ok

File: README.md [hunk 4]
```diff
@@ -71,46 +69,66 @@ kubectl get pods -n istio-system
...
+
 ```
 nginmesh-0.6.0/install/kafka/install.sh
 ```
 
-3. Set up nginmesh topic by running following script
+Note: In GKE environment you may need to grant permission to default serviceaccount for cluster-wide access:
+
+```
+kubectl create serviceaccount --namespace kube-system tiller
```
Comment (sehz) [inline README.md:100]: this should be consolidated instead of repeat

**Review by sehz — APPROVED** [review]

**Pull request merged**